R/zzz.R

```diff
@@ -42,7 +42,7 @@ cr_GET <- function(endpoint, args, todf = TRUE, on_error = warning, parse = TRUE
       get_route(res), get_err(res)), call. = FALSE)
     list(message = NULL)
   } else {
-    stopifnot(res$response_headers$`content-type` == "application/json;charset=UTF-8")
```

Comment:
Given the implementation in get_err() which contains

x$response_headers$`content-type` == "application/json;charset=UTF-8" ||
    x$response_headers$`content-type` == "application/json"

I would rather suggest:

stopifnot(res$response_headers$`content-type` == "application/json;charset=UTF-8" ||
              res$response_headers$`content-type` == "application/json")